src/Symfony/Bridge/PhpUnit/Legacy/CoverageListenerTrait.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\PhpUnit\Legacy;
+
+use PHPUnit\Framework\Test;
+
+/**
+ * PHP 5.3 compatible trait-like shared implementation.
+ *
+ * @author Grégoire Pineau <lyrixx@lyrixx.info>
+ *
+ * @internal
+ */
+class CoverageListenerTrait
+{
+    private $sutFqcnResolver;
+
+    public function __construct(callable $sutFqcnResolver = null)
+    {
+        $this->sutFqcnResolver = $sutFqcnResolver;
+    }
+
+    public function startTest($test)
+    {
+        $annotations = $test->getAnnotations();
+
+        if (isset($annotations['class']['covers']) || isset($annotations['method']['covers'])) {
+            return;
+        }
+
+        $sutFqcn = $this->findSutFqcn($test);
+        if (!$sutFqcn) {
+            return;
```

Comment:
those would have `@coversNothing`.
so you will not hit this line for them

---------------------------------------------------------------

at least please put option to control this behaviour instead of silencing not-found scenario. as while using the listener you have no obvious way to see that src class was not found for given test class